libproxy/url.cpp

```diff
@@ -26,6 +26,7 @@
 #include <sstream> // For int/string conversion (using stringstream)
 #include <cstdio>  // For sscanf()
 #include <cstdlib>    // For atoi()
+#include <cerrno>  // For errno and EINTR
```

Comment:
In all other sources files where needed we have `#include <errno.h>` and I'd prefer to have the include statement aligned across the source files, not using errno.h in one place and cerrno in other areas

Reply:
OK, but the PR was just merged. Should I create new one?

Reply:
> OK, but the PR was just merged. Should I create new one?

I rather meant to align things - in c++, <cerrno> is actually the preferred method imho

Reply:
> I rather meant to align things - in c++, is actually the preferred method imho

Ahh, OK. Yes, in C++ it's better to use the C++ headers.